src/core/gosling.schema.ts

```diff
@@ -209,6 +209,29 @@ export interface CommonTrackDef extends CommonViewDef {
 export interface DataTrack extends CommonTrackDef {
     data: DataDeep;
 }
+/**
+ * A placeholder track. In contrast to other tracks, this track does not display any data. Instead it provides
+ * empty space for third party tools to display their data on top of.
+ */
+export interface DummyTrack extends CommonTrackDef {
+    /** Used to specify the dummy track */
+    type: 'dummy-track';
+    /** Defines how the track is styled */
+    style?: DummyTrackStyle;
+}
+
+export interface DummyTrackStyle {
+    /** Background color of the track */
+    background?: string;
+    /** Specify the font size of the title */
+    textFontSize?: number;
+    /** Specify the font weight of the title. */
+    textFontWeight?: 'bold' | 'normal';
+    /** Specify the stroke color of title. */
+    textStroke?: string;
+    /** Specify the stroke width of the title. */
+    textStrokeWidth?: number;
```

Comment:
I believe these are all of the styles we talked about. Let me know if there are any more you think would be useful!